extensions/amp-render/1.0/amp-render.js

```diff
@@ -113,6 +113,21 @@ function getAmpScriptJson(ampdoc, src) {
     });
 }
 
+/**
+ * @param {!AmpElement} element
+ * @param {boolean} shouldRefresh true to force refresh of browser cache.
+ * @return {!Object} options object to pass to `batchFetchJsonFor` method.
+ */
+function buildOptionsObject(element, shouldRefresh = false) {
+  return {
+    xssiPrefix: element.hasAttribute('xssi-prefix')
+      ? element.getAttribute('xssi-prefix')
+      : undefined,
```

Comment:
Does it suffice to allow `xssiPrefix` to default to `null` if the attribute isn't provided? 
```suggestion
    xssiPrefix: element.getAttribute('xssi-prefix'),
```